src/main/kotlin/lotto/core/WinningLotto.kt

```diff
@@ -0,0 +1,15 @@
+package lotto.core
+
+import lotto.core.constant.LottoConstants
+
+class WinningLotto(private val lotto: Lotto, private val bonusNumber: LottoNumber) {
+    constructor(numbers: List<LottoNumber>, bonusNumber: LottoNumber) : this(Lotto(numbers), bonusNumber)
+
+    init {
+        require(!lotto.contains(bonusNumber)) { LottoConstants.ERROR_INVALID_BONUS_NUMBER }
```

Comment:
contains 메서드를 `operator` 메서드로 정의해준다면 아래와 같이 변경해볼 수도 있을 것 같아요. 😉

```suggestion
        require(bonusNumber !in lotto) { LottoConstants.ERROR_INVALID_BONUS_NUMBER }
```

Reply:
네 좋은 방법인거 같습니다. 생각못했네요.
반영하겠습니다.